### PR TITLE
[RM] Fix 2 anonymous hyperlink mismatches

### PR DESCRIPTION
**Pull request by collivier**
It basically forbids rendering any document.
See "backrefs" attribute for IDs.

Signed-off-by: Cédric Ollivier <cedric.ollivier@orange.com>

### CODE REVIEW

File: doc/ref_model/chapters/chapter03.rst [hunk 1]
```diff
@@ -110,7 +110,7 @@ A virtual machine and container can consume storage through a number of means. T
 
 -  managed via the hypervisor and container runtime (Hypervisor Attached for virtual machine and Container Persistent for containers) and is connected via cloud infrastructure underlay network and
 -  Shared File Storage and the Object storage which is connected via the tenant / user overlay network.
-   The details of the tenant storage consumption model are covered in section `"Storage for Tenant Consumption"`__.
+   The details of the tenant storage consumption model are covered in section `Storage for Tenant Consumption <./chapter03.md#3.6.3>`__.
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter03.rst:113]: This is an internal reference within the chapter and so I suggest:
```suggestion
   The details of the tenant storage consumption model are covered in section `Storage for Tenant Consumption`__.
```

File: doc/ref_model/chapters/chapter03.rst [hunk 2]
```diff
@@ -226,7 +226,7 @@ Improve cost/performance               Better cost and less power/cooling by imp
 
 Hardware Accelerators can be used to offload software execution for purpose of accelerating tasks to achieve faster performance, or offloading the tasks to another execution entity to get more predictable execution times, efficient handling of the tasks or separation of authority regarding who can control the tasks execution.
 
-More details about Hardware Acceleration are in `Hardware Acceleration Abstraction`__.
+More details about Hardware Acceleration are in `Hardware Acceleration Abstraction <./chapter03.md#3.8>`__.
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter03.rst:229]: Similar to previous:
```suggestion
More details about Hardware Acceleration are in `Hardware Acceleration Abstraction`__.
```

**Comment by pgoyal01** [comment]
@collivier I realise that my ask is not related to the PR but there is an issue at line 591: "**3.5.6** Time Sensitive Networking" -- the section number needs to be deleted

**Comment by pgoyal01** [comment]
The external references like in line 124: "`Storage Extensions <./chapter04.md#4.2.6>`__ section" should be changed to: "`Storage Extensions <./chapter04.md#storage-extensions>`__ section"

**Comment by collivier** [comment]
@pgoyal01 Thank you for the review. This ref is being modified in a bigger change removing all obsolete md refs (I added a second commit to share the work in progress). The first proposal you commented was a crucial quick fix for an issue which breaks everything (and which is very difficult to detect). But I'm finishing the second commit before the first one is reviewed.

**Review by collivier — COMMENTED** [review]
It must be double checked both by RM team! It's an early share to highlight all the issues.

**Review by karinesevilla — COMMENTED** [review]
@collivier For chapter 7, all external links are good.
There as still 3 internal links to fix:
Sections: Root of trust for measurements(RTM), 7.6.1 and Zero trust architecture, 7.6.1 and 7.6.5

**Review by gkunz — APPROVED** [review]

**Review by pgoyal01 — APPROVED** [review]

**Review by walterkozlowski — APPROVED** [review]

**Pull request merged**